consul/README.md

```diff
@@ -149,6 +149,34 @@ Optionally, you can configure Consul to also send data to the Agent through [Dog
 
 3. [Restart the Agent][6].
 
+#### OpenMetrics
+
+Instead of using DogStatsD, you can use the `use_prometheus_endpoint` to get the same metrics. 
+
+
+**Note**: Use either the DogStatsD method or the OpenMetrics method,  do not enable both for the same instance or there will be duplicated metrics.
```

Comment:
```suggestion
**Note**: Use either the DogStatsD method or the OpenMetrics method, do not enable both for the same instance or there will be duplicated metrics.
```

Reply:
Can we detect this and throw a configuration error?

Reply:
I will try to do that 👍